helm/operational-load-exporter/templates/policy-exception.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.kyvernoPolicyExceptions.enabled }}
+  {{- if .Capabilities.APIVersions.Has "policy.giantswarm.io/v1alpha1/PolicyException" -}}
```

Comment:
In which scenario we need this check for this app?

If the app is optional, and is installed after the cluster has been deployed, then the PolicyException CRD should be there (if it's not it's an issue that would be hidden with this check).

If the app is deployed during cluster creation (one of default apps), then this would mean that the app could be installed without its PolicyException, which I don't think we should do (IMO better let it fail and succeed eventually).

cc @stone-z should we ever do these checks for PolicyException CRD when installing apps?

Reply:
I think that most importantly we need to question if the app couldn't be adapted to conform with the PSS restrictions and avoid adding exceptions.

In this case I don't see why the deployment can't meet those policies.

Reply:
so I added that check after asking shield. I think it allows the transition to work. It can be deployed now in the WC cluster that does not have PolicyException yet and transition to that state. It can be used for someone else that does not run our policy API

Reply:
I am with you Mate, we could potentially remove that exceptions. @JosephSalisbury we can pair together for an hour and see if it is doable

Reply:
This is a WC app so IMO should not have the capability check.
Also agree, looking at the exceptions here I don't think this app actually needs them to run. Better just to fix the `securityContext`

Reply:
Also, if you DO need exceptions for this app, please align the toggle with https://github.com/giantswarm/giantswarm/issues/28341

Reply:
Reviewing the chart I see the problem is not really an operational load exporter since it could run with the current PSS implementation but there is a side container, grafana agent, that forwards the metric to grafana cloud and this container needs those high level permissions 🤷